examples/gwNodeInt/RF24GatewayNodeInt.cpp

```diff
@@ -83,11 +82,9 @@ int main(int argc, char** argv)
             mesh.begin();
         }
 
-        gw.interrupts(); // Re-enable interrupts when done accessing the radio
```

Comment:
This kind of API change implies a feature (aka minor) version bump. We should probably add something to CMakeLists.txt that checks RF24 lib version. CMake's [`find_library()`](https://cmake.org/cmake/help/latest/command/find_library.html) doesn't seem to care about the found lib's version, so I might have to dig deeper into how to mandate a certain RF24 version -- `find_library()`'s `VALIDATOR` arg (new in CMake v3.25) seems like a strong lead.

BTW, I was thinking this new Linux kernel IRQ support would be released under RF24 v1.5.0.

Reply:
Well most users should be using the installer, using the latest versions, but I see where you're going and its probably a good idea. Although these changes should still work with previous RF24 versions in theory.

Reply:
CMake's [`find_package()`](https://cmake.org/cmake/help/latest/command/find_package.html) has a [version verification](https://cmake.org/cmake/help/latest/command/find_package.html#config-mode-version-selection) if a required version is specified! 👀 I'm not sure if that will work with RF24 as a shared library though.